pages/portfolio.html

```diff
@@ -1,12 +1,44 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+<meta charset="UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Document</title>
+<link href="../styles/style.css" rel="stylesheet" />
 </head>
 <body>
-  
+    <div class="container">
+        <div class="header">
+        
+        </div>
+        <div class="sidebar">
+            <img class="img-side" src="../images/dubai.jpeg" alt="my image">
+        </div>
+        <div class="content">
+          <nav>
+            <h2> <a href="index.html">Home</a></h2> <h2 class="about"> <a href="about.html">About me</a></h2>
+          </nav>
+                <br>
+                <br>
+                <h1> Projects:</h1>
+                <nav>
+                <ul>
+                  <li><h2><a href="https://github.com/Intesarsiraj/viewing-party">Viewing Party</a> </h2></li><br>
```

Comment:
The `br` tag should not be used to create space between the items in the list. There are ways to make that space happen using CSS. This causes errors in the HTML validator.